metadata-models/src/main/resources/entity-registry.yml

```diff
@@ -7,6 +7,8 @@ entities:
       - subTypes
       - datasetProfile
       - datasetUsageStatistics
+      - datasetExpectations
```

Comment:
Do we need both of these? I think we can probably start with a single datasetValidation aspect